### PR TITLE
Assignment2 @Matt Pennock

### PR DESCRIPTION
**Pull request by DakotaCondos**
Website Link: https://yellow-hill-0209e201e.1.azurestaticapps.net

### CODE REVIEW

**Comment by tnadalini** [comment]
Instructions

Note: Custom CSS can only be used where specified.

    Create a custom component to show available words ✅

    Add a question mark key to the keyboard. This represents a placeholder for any character ✅
    Create/extend a word list component with a method called validWords that takes a string with question marks and returns an array of valid words ✅
    Create unit tests for the validWords method ✅
    Display the number of valid words based on the entered letters ✅
    When the number is clicked, show a dialog with all the available words with scrolling if there are too many to show on the screen. ✅
    When a word is clicked, the word should be entered as the guess but not submitted ✅
    Add a hover over for the word so it is obvious that it can be clicked. (No CSS) ✅

    Update the App Bar ✅

    Name of the app on the left with an icon ✅
    Choose an icon for the app (from Material Design or somewhere on the web) ✅
    Clicking the name and icon of the app take you to the home page ✅
    Hamburger menu on the right ✅
    Menu option for an About page ✅
        Create an About page that has a few sentences about this class project and how great Meg is ✅
        Make sure this has a back button ✅

    Add a settings dialog ✅

    Add a settings selection to the App Bar's menu with a gear icon. This should open this dialog regardless of where you are on the site ✅
    Option for dark and light mode ✅
    Develop two additional color schemes (with creative names) and allow the user to change to them. Schemes should look good in both light and dark mode ✅
    The above two items must be implemented with built in Veutify features ✅

    Style the main game page ✅

    Make it look like [wordle.com](https://www.nytimes.com/games/wordle/index.html) or the Wordle mobile app ✅
    Add gradient to guesses blocks and keys (with CSS) ✅
    Add drop shadow to keys (No CSS) ✅
    Make the app responsive (No CSS) ✅

    Include a link to your Azure App in your PR comments.

**Comment by linhtran8699** [comment]
The website looks great. The color and the theme are awesome. 
However, when I played the game and lost, the hidden word wasn't actually shown. The message was "You lost... :( The word was ${this.word}". 
Also, it's just my own reference, but when I click on the hamburger icon, and it pops up home, game, about; I would prefer them to not cover my hamburger button. You can use something like "offset-y" in your v-menu. 
And again, great job on the colors and the theme, I love it!

**Comment by DakotaCondos** [comment]
> The website looks great. The color and the theme are awesome. However, when I played the game and lost, the hidden word wasn't actually shown. The message was "You lost... :( The word was ${this.word}". Also, it's just my own reference, but when I click on the hamburger icon, and it pops up home, game, about; I would prefer them to not cover my hamburger button. You can use something like "offset-y" in your v-menu. And again, great job on the colors and the theme, I love it!

Thanks for the review. Was not aware it was doing the ${this.word}

**Comment by linhtran8699** [comment]
> > The website looks great. The color and the theme are awesome. However, when I played the game and lost, the hidden word wasn't actually shown. The message was "You lost... :( The word was ${this.word}". Also, it's just my own reference, but when I click on the hamburger icon, and it pops up home, game, about; I would prefer them to not cover my hamburger button. You can use something like "offset-y" in your v-menu. And again, great job on the colors and the theme, I love it!
> 
> Thanks for the review. Was not aware it was doing the ${this.word}

sure. I am glad I could help!

**Comment by hewrtarkhany** [comment]
nice visual and shadowing on the keyboard and the screen

File: wordle-web/components/keyboard.vue [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+@Component
+export default class KeyBoard extends Vue {
+  @Prop({ required: true })
+  wordleGame!: WordleGame
+
+  showWords!: ShowWords
+
+  watchProperty: boolean = true
+  watchNumber: number = 631
+  // buttonSound = useSound('~/assets/button.mp3')
```
Comment (mmwoodfo) [inline wordle-web/components/keyboard.vue:56]: `buttonSound = new Audio('...file here...')`

**Review by mmwoodfo — COMMENTED** [review]
Looks great guys!

**Pull request closed without merging**